# Double: Selected pose publication rate
# Units: Hz
pose_pub_rate: 10.0

# Double: NDT fitness score threshold above which the result can be considered degraded
fitness_score_degraded_threshold: 20.0

# Double: NDT fitness score threshold to determine if the result can be considered unusable
fitness_score_fault_threshold: 100000.0

# Double: NDT result frequency threshold below which the result can be considered degraded
# Units: Hz
ndt_frequency_degraded_threshold: 8.0

# Double: NDT result frequency threshold below which the result can be considered unusable
# Units: Hz
ndt_frequency_fault_threshold: 0.01

# Integer: The timeout of an auto initialization system
# Units: ms
auto_initialization_timeout: 30000

# Integer: Maximum allowed GNSS only operation for the localization system while in AUTO mode.
# Units: ms
gnss_only_operation_timeout: 20000

# Integer: Maximum allowed number of sequential timesteps to let lidar initialize before switching to GPS only mode
# NOTE: Only used in GNSS only with NDT initialization mode
# Units: Int
sequential_timesteps_until_gps_operation: 5

# Integer Enum: Localization mode for the platform
# 0 - NDT only
# 1 - GNSS only
# 2 - AUTO select between NDT and GNSS with GNSS timeout
# 3 - AUTO select between NDT and GNSS without GNSS timeout
# 4 - GNSS only with NDT initialization
localization_mode: 4
